testsuite/gnat2goto/tests/private_type_decl/test.out

```diff
@@ -0,0 +1,2 @@
+[1] file private_type_decl.adb line 8 assertion: SUCCESS
```

Comment:
Is there still a `test.opt` file in this folder?

Reply:
There should not be test.opt in the folder.  
In folder testsuite/gnat2goto/tests/private_type_dec, in my local repository I do not seem to have a test.py file.  Obviously there should be one. I have added one on my machine (not pushed) and the test runs ok.